vtexdocs/RapiDoc
# Fix/Schema Tabs

#### What is the purpose of this pull request?

To improve usability of API Reference pages in the Developers Portal by renaming the schema tab (the one that shows the parameters in an API request/response) and making it the default tab instead of the example one.

#### What problem is this solving?

Developers Portal's users are not being able to find the parameters for an API request/response easily.

#### How should this be manually tested?

Go to [this page](https://deploy-preview-151--elated-hoover-5c29bf.netlify.app/docs/api-reference/antifraud-provider-protocol#post-/transactions) and check that the schema tab (in the Request Body and Response sections) is now named "Parameters" and is active by default when you open the page (and the example tab is not). Try testing other documentations as well.

#### Types of changes
